packages/world/src/systems/init/InitInteractablesSystem.sol

```diff
@@ -97,5 +98,14 @@ contract InitInteractablesSystem is System {
       1
     );
     createDoubleInputWithStationRecipe(WorkbenchObjectID, StoneObjectID, 4, SilverBarObjectID, 1, PipeObjectID, 1);
+    createDoubleInputWithStationRecipe(
+      ThermoblasterObjectID,
```

Comment:
dang spawn tiles are a luxury hehe

(its fine since this is a placeholder for now, will tune the recipe later)